client/app/telecom/telephony/alias/configuration/changeType/telecom-telephony-alias-configuration-changeType.html

```diff
@@ -1,104 +1,127 @@
-
-                    <button type="submit"
-                            class="btn btn-primary"
-                            data-ng-disabled="!changeTypeForm.$valid || AliasConfigurationChangeTypeCtrl.loading.changing || !AliasConfigurationChangeTypeCtrl.number.feature.hasChange()"
-                            data-translate="submit">
-                    </button>
-                </form>
-                <div class="text-center"
-                     data-ng-if="AliasConfigurationChangeTypeCtrl.loading.changing">
-                    <oui-spinner></oui-spinner>
-                    <h4 data-translate="telephony_alias_config_change_type_doing"></h4>
-                </div>
+        <div class="row w-100 d-md-flex">
+            <div class="col-lg-4 col-sm-12 mb-4">
+                <oui-select-picker class="h-100 pr-0 ml-0"
+                                   data-match="name"
+                                   data-model="$ctrl.selectedFeatureType"
+                                   data-label="{{:: 'telephony_alias_config_change_type_label_cloudIvr' | translate }}"
+                                   data-values="[{ id: 'cloudIvr', name: $ctrl.$translate.instant('telephony_alias_config_change_type_desc_cloudIvr') }]"
+                                   data-variant="light">
+                    <span class="transclude-image-container">
+                        <img src="assets/images/aliasFeature/schema-cloudIvr.svg"
+                             alt="schema-cloudIvr.svg">
+                    </span>
+                </oui-select-picker>
             </div>
-            <div class="col-xs-12 col-md-6 widget-presentation">
-                <header class="widget-presentation-header">
-                    <h2 class="widget-presentation-title"
-                        data-translate="telephony_alias_config_change_type_desc">
-                    </h2>
-                </header>
-                <p data-ng-if="AliasConfigurationChangeTypeCtrl.number.feature.featureType"
-                   data-translate="{{'telephony_alias_config_change_type_desc_' + AliasConfigurationChangeTypeCtrl.number.feature.featureType}}">
-                </p>
+            <div class="col-lg-4 col-sm-12 mb-4">
+                <oui-select-picker class="h-100 pr-0"
+                                   data-match="name"
+                                   data-model="$ctrl.selectedFeatureType"
+                                   data-label="{{:: 'telephony_alias_config_change_type_label_conference' | translate }}"
+                                   data-values="[{ id: 'conference', name: $ctrl.$translate.instant('telephony_alias_config_change_type_desc_conference') }]"
+                                   data-variant="light">
+                    <span class="transclude-image-container">
+                        <img src="assets/images/aliasFeature/schema-conference.svg"
+                             alt="schema-conference.svg">
```

Comment:
Could be more descriptive?

Reply:
https://github.com/ovh-ux/ovh-manager-telecom/pull/981#discussion_r225066859